analyzers/src/SonarAnalyzer.CSharp/Rules/ExceptionsShouldBeLoggedOrThrown.cs

```diff
@@ -67,4 +67,32 @@ public sealed class ExceptionsShouldBeLoggedOrThrown : SonarDiagnosticAnalyzer
                          SyntaxKind.CatchClause);
                 }
             });
+
+    private sealed class LoggingInvocationWalker(SemanticModel model) : CatchLoggingInvocationWalker(model)
+    {
+        public override void VisitParenthesizedLambdaExpression(ParenthesizedLambdaExpressionSyntax node)
+        {
+            // Skip processing to avoid false positives.
+        }
+
+        public override void VisitSimpleLambdaExpression(SimpleLambdaExpressionSyntax node)
+        {
+            // Skip processing to avoid false positives.
+        }
```

Comment:
Why not leave the two lambda boys on the base walker?
I think we don't want to visit them in any of the rules, since we don't symbolically execute.